source/agora/test/NetworkManager.d

```diff
@@ -136,20 +137,29 @@ unittest
         }
     }
 
-    TestConf conf = { topology : NetworkTopology.OneValidator, nodes : 3 };
+    TestConf conf = {
+        topology : NetworkTopology.MinimumValidators,
+        nodes : NetworkTopology.MinimumValidators + 2
+    };
     auto network = makeTestNetwork!BadAPIManager(conf);
     network.start();
     scope(exit) network.shutdown();
     scope(failure) network.printLogs();
     network.waitForDiscovery();
 
     auto nodes = network.clients;
-    auto node_validator = nodes[0];  // validator, creates blocks
-    auto node_test = nodes[1];  // full node, does not create blocks
-    auto node_bad = nodes[2];  // full node, returns bad blocks in getBlocksFrom()
+    assert(nodes.length >= Enrollment.MinValidatorCount+2);
+    // validator, creates blocks
+    auto node_validator = nodes[0];
```

Comment:
Is `node_validator` now unused? Then it can be removed.

Reply:
Ah I see it's used below in:

```
assert(node_validator.getBlockHeight() == 10,
```

But maybe this should test for all validators instead?